scalafmt-tests/src/test/resources/unit/Source3.source

```diff
@@ -0,0 +1,12 @@
+<<< kind projector #2560 
```

Comment:
optional: we could put it in one of existing scala3 files, or `Dialect.stat`.

Reply:
Since it's more related to Scala 2, I put it into `unit/Type.stat`